pkg/config/settings/runtime_setting_log_level.go

```diff
@@ -38,16 +43,53 @@ func (l LogLevelRuntimeSetting) Get() (interface{}, error) {
 	if err != nil {
 		return "", err
 	}
-	return level.String(), nil
+
+	var source log.LogLevelSource
+	source, err = log.GetSource()
+	if err != nil {
+		return "", err
+	}
+
+	logLevelStatus := LogLevelStatus{
+		Level:  level.String(),
+		Source: source,
+	}
+	return logLevelStatus.Level, nil
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** Let's not change it as it's not needed‏
```suggestion
	return level.String(), nil
```